pep257.py

```diff
@@ -377,8 +377,15 @@ def parse_all(self):
             elif (self.current.kind == tk.STRING or
                   self.current.value == ','):
                 all_content += self.current.value
+            elif self.current.kind == tk.NAME:
+                all_content += "'%s'" % self.current.value
```

Comment:
I believe this has been proposed before, and rejected.  I cant seem the find the issue now in any of the relevant projects.
While using symbols in __all__ may sometimes not cause a syntax error, it also doesnt work correctly.  They need to be a string.